src/ActionDropdown/ActionDropdown.tsx

```diff
@@ -56,6 +57,7 @@ export const ActionDropdown: FC<Props> = ({
       <OuterContainer open={open}>
         <List options={list} selectOption={onSelect} />
       </OuterContainer>
+      <FontStyle />
```

Comment:
Should we load the font-face declarations into each component or add them globally?

Reply:
Happy with either!

Reply:
Added globally and removed font-family declarations on component level